packages/payments-client/src/components/pages/payments/index.tsx

```diff
@@ -0,0 +1,281 @@
+import React from 'react'
+import uuidv4 from 'uuid/v4'
+import useSWR, { mutate } from 'swr'
+import { useHistory, useLocation } from 'react-router'
+import { Field } from 'formik'
+import { History } from 'history'
+import ErrorBoundary from '@/components/hocs/error-boundary'
+import { cleanObject, errorMessages } from '@reapit/utils'
+import {
+  Pagination,
+  Table,
+  Loader,
+  setQueryParams,
+  H3,
+  toLocalTime,
+  isEmptyObject,
+  Section,
+  Alert,
+  Button,
+  Checkbox,
+  Formik,
+  Form,
+  notification,
+  DATE_TIME_FORMAT,
+} from '@reapit/elements'
+import Routes from '@/constants/routes'
+import PaymentsFilterForm, { PaymentsFilterFormValues } from '@/components/ui/payments/payments-filter-form'
+import PaymentRequestModal from '@/components/ui/payments/payment-request-modal'
+import { URLS } from '../../../constants/api'
+
+interface CustomerAddress {
+  buildingName: string
+  buildingNumber: string
+  countryId: string
+  line1: string
+  line2: string
+  line3: string
+  line4: string
+  postcode: string
+}
+
+interface Customer {
+  email: string
+  forename: string
+  homePhone: string
+  id: string
+  mobilePhone: string
+  name: string
+  primaryAddress: CustomerAddress
+  surname: string
+  title: string
+  type: string
+}
+
+interface Payment {
```

Comment:
Can you add the payments swagger doc to @reapit/foundations-ts-definitions please? Just add to the schema here https://github.com/reapit/foundations/blob/master/packages/foundations-ts-definitions/scripts/fetch-definitions.js#L59 and run yarn fetch-definitions in the package